wave_kinematics/include/wave/kinematics/two_wheel.hpp

```diff
@@ -0,0 +1,40 @@
+/** @file
+ * @ingroup kinematics
+ */
+
+#ifndef WAVE_KINEMATICS_TWOWHEEL_HPP
+#define WAVE_KINEMATICS_TWOWHEEL_HPP
+
+#include <float.h>
+#include <iostream>
+
+#include "wave/utils/utils.hpp"
+
+namespace wave {
+/** @addtogroup kinematics
+ *  @{ */
+
+/** Generic two wheel robot motion model */
```

Comment:
Two wheel robot: https://www.google.ca/search?q=two+wheel+robot&source=lnms&tbm=isch
It may be more clear to call this a "bicycle model"